src/transport.js

```diff
@@ -0,0 +1,44 @@
+const nodemailer = require('nodemailer')
+const ses = require('nodemailer-ses-transport')
+const sendgrid = require('nodemailer-sendgrid-transport')
+const postmark = require('nodemailer-postmark-transport')
+const mailgun = require('nodemailer-mailgun-transport')
+const stub = require('nodemailer-stub-transport')
+const winston = require('winston')
+
+module.exports = env => {
+  switch(env('TRANSPORT')) {
+    case 'ses':
+      AWS.config.update({
+        accessKeyId: env('AWS_KEY'),
+        secretAccessKey: env('AWS_SECRET'),
+        region: env('AWS_REGION')
+      })
+      return nodemailer.createTransport(ses({ ses: new AWS.SES() }))
+    case 'sendgrid':
+      return nodemailer.createTransport(sendgrid({
+        auth: {
+          api_user: env('SENDGRID_USERNAME'),
+          api_key: env('SENDGRID_PASSWORD') || env('SENDGRID_API_KEY')
```

Comment:
Sendgrid does actually allow for username/password, or just api key. I'll take out the username and password though :)